### DISCUSSION TITLE
Can't run scans

### DISCUSSION THREAD

**Discussion by alaxalves** [aquasecurity/starboard#1172] [Questions & Help] [answered]
I have just installed starboard, and I'm trying to check the scans out in Octant using the plugin. But I'm running `kubectl starboard scan vulnerabilityreports deployment/example` command and I gets me the error with ANY of my deployments:

```
E0502 16:51:39.894236   29907 runnable_job.go:162] Container dasa terminated with OOMKilled: 
error: running scan job: warning event received: Job has reached the specified backoff limit (BackoffLimitExceeded)
```
What am I missing here please?

**Comment by alaxalves** [comment]
FYI: My deployments use images from a private registry and the generated secrets are correct. Starboard CLI and starboard-operator on v0.15.4

**Comment by alaxalves** [comment]
@danielpacak In case you missed it :)

**Reply by danielpacak** [reply]
Hi @alaxalves 👋 

TBH I'm confused what are doing. Starboard Operator & Starboard CLI use the same CRDs but they're supposed be used separately.

Anyway, the OOMKilled error might indicate that a scan job that runs Trivy executable binary reached the resource limit and was killed by the orchestrator. You can confirm that by increasing the resource limits in the `starboard-trivy-config` ConfigMap that is created in the installation namespace. Look for the for following configuration properties:

```
trivy.resources.requests.cpu: 100m
trivy.resources.requests.memory: 100M
trivy.resources.limits.cpu: 500m
trivy.resources.limits.memory: 500M
```

**Reply by alaxalves** [reply]
> Hi @alaxalves wave
> 
> TBH I'm confused what are doing. Starboard Operator & Starboard CLI use the same CRDs but they're supposed be used separately.
> 
> Anyway, the OOMKilled error might indicate that a scan job that runs Trivy executable binary reached the resource limit and was killed by the orchestrator. You can confirm that by increasing the resource limits in the `starboard-trivy-config` ConfigMap that is created in the installation namespace. Look for the for following configuration properties:
> 
> ```
> trivy.resources.requests.cpu: 100m
> trivy.resources.requests.memory: 100M
> trivy.resources.limits.cpu: 500m
> trivy.resources.limits.memory: 500M
> ```

Hi @danielpacak thanks for the reply.
I have increased those limits, and had the same error output. I have also double checked the credendials that are being used in the job, and they're correct. 
BTW I have starboard operator running in my K8s cluster for a while now and I was trying to configure the octant plugin in order to have some sort of UI for starboard operator. This required me to install the CLI too to trigger some scans that could be shown in the octant UI, and that's what caused the error I described here.

**Reply by danielpacak** [reply]
If you install everything that's available it's going to be hard to find the root cause. I suggest you start just with Starboard Operator and adjust or comment out resource limits to narrow down the OOMKilled issue and see if the problem persists. Note also that the Starboard Operator and Starboard CLI have their own ConfigMaps to configure Trivy plugin so the limits must be set in both places.

**Reply by alaxalves** [reply]
Okay, Will do that!
Thanks @danielpacak I'll report here the results.

**Comment by alaxalves** [comment, marked as answer]
Hi @danielpacak thanks for the tips. I have managed to use both operator and CLI, it was resource limit indeed, I was struggling to get the proper values. Thanks a lot!
